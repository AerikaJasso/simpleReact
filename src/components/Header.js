import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => (
  <header>
    <h1>Expensify</h1>
    <ul>
      <NavLink to="/" activeClassName="is-active" exact={true}>
        <li>Dashboard</li>
      </NavLink>
      <NavLink to="/create" activeClassName="is-active">
        <li>Create Expense</li>
      </NavLink>
      
    </ul>
  </header>
);

export default Header;