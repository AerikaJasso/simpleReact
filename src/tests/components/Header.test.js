// react-test-render
import React from 'react';
import { shallow, mount, render } from 'enzyme';
import Header from '../../components/Header';

test('should render Header correctly', () => {
  const wrapper = shallow(<Header />);
  
  expect(wrapper).toMatchSnapshot();
  // expect(wrapper.find('h1').text()).toBe('Expensify');
  // const renderer = new ReactShallowRenderer();
  // renderer.render(<Header />);

  // expect(renderer.getRenderOutput()).toMatchSnapshot();
});