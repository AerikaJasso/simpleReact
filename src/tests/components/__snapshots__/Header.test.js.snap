// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <ul>
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      exact={true}
      to="/"
    >
      <li>
        Dashboard
      </li>
    </NavLink>
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      to="/create"
    >
      <li>
        Create Expense
      </li>
    </NavLink>
  </ul>
</header>
`;
